boards/arm/gd32f450i_eval/gd32f450i_eval-pinctrl.dtsi

```diff
@@ -11,4 +11,12 @@
 			pinmux = <USART0_TX_PA9>, <USART0_RX_PA10>;
 		};
 	};
+
+	spi5_default: spi5_default {
+		pins1 {
+			pinmux = <SPI5_SCK_PG13>,  <SPI5_NSS_PG8>,   \
+				 <SPI5_MOSI_PG14>, <SPI5_MISO_PG12>, \
+				 <SPI5_IO2_PG10>,  <SPI5_IO3_PG11>;
```

Comment:
nit: `\` not needed:
```
			pinmux = <SPI5_SCK_PG13>, <SPI5_NSS_PG8>,
				 <SPI5_MOSI_PG14>, <SPI5_MISO_PG12>,
				 <SPI5_IO2_PG10>, <SPI5_IO3_PG11>;
```

are IO2/IO3 required? (only needed for quad mode, as far as I know). Depending on the SPI speed you may need to set `slew-rate`.

Reply:
> nit: `\` not needed:

Thanks for you advice, removed.

> are IO2/IO3 required? (only needed for quad mode, as far as I know). Depending on the SPI speed you may need to set `slew-rate`.

IO2/IO3 are not required, no quad spi driver for it now, remove it.